cmd/font/main.go

```diff
@@ -36,6 +26,13 @@ func main() {
 		os.Args = os.Args[1:]
 	}
 
+	cmds := map[string]func(*sfnt.Font) error{
+		"scrub":    Scrub,
+		"info":     Info,
+		"stats":    Stats,
+		"metrics":  Metrics,
+		"features": Features,
+	}
```

Comment:
Oh, this was intentionally at the top, so usage() could eventually use it. No worries for now.

Reply:
I was thinking of doing the same thing. I was gonna do it later, so I just simplified the code until then. It can easily be moved around.